contracts/RestrictedNORI.sol

```diff
@@ -772,33 +776,30 @@ contract RestrictedNORI is
    * @dev schedules are created when removal tokens are listed for sale in the market contract,
    * so this should only be invoked during `tokensReceived` in the exceptional case that
    * tokens were sent to this contract without a schedule set up.
+   *
+   * Note that revert strings are used instead of custom errors here for proper surfacing
+   * from within the market contract `onERC1155BatchReceived` hook.
    */
   function _createSchedule(uint256 projectId) internal {
     Removal.ScheduleData memory scheduleData = _removal
       .getScheduleDataForProjectId(projectId);
     if (scheduleData.startTime == 0) {
-      revert InvalidScheduleStartTime({projectId: projectId});
+      revert("InvalidScheduleStartTime");
```

Comment:
does `require(scheduleData.startTime != 0, "InvalidScheduleStartTime")` work as well? I think I'd prefer that syntax in these cases